.github/workflows/generate-docs.yml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2024 The Nuclio Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+name: Check nuctl docs
+
+on:
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  check-nuctl-docs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.21'
+
+      - name: Generate Nuctl docs
+        run: make generate-nuctl-docs
+
+      - name: Check for changes in docs
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
```

Comment:
These values can be stored in env vars (see [this](https://github.com/nuclio/nuclio/blob/development/.github/workflows/ci.yaml#L32-L40))

Reply:
@TomerShor added